RoomsQuickstart/Program.cs

```diff
@@ -35,9 +54,10 @@ static async Task Main(string[] args)
                 {
                     await CreateRoom();
                     await UpdateRoom(rooms[0]);
-                    await UpdateParticipants(rooms[0]);
-                    await AddParticipants(rooms[0], addeddParticipant);
-                    await RemoveParticipants(rooms[0], addeddParticipant);
+                    await GetRoom(rooms[0]);
+                    await ListRoom();
+                    await AddOrUpdateParticipants(rooms[0], addedParticipant);
```

Comment:
wondering if you are running into 409 conflict as those calls are running concurrently. I have to add delays in js quickstart to avoid the error.